### DISCUSSION TITLE
Provincial Data Updating

### DISCUSSION THREAD

**Discussion by lizelgreyling** [dsfsi/covid19za#971] [Q&A]
Hi - I am having a serious problem getting the Mpumalanga district data from the Mpumalanga Department of Health. They have simply stopped updating the data in February 2022 and are ignoring all communication from me. This means that the Provincial data for Mpumalanga on this repository has no way of being updated at the moment. Does anyone have any contact or some angle that I can use to get put pressure on them?

**Comment by shaze** [comment]
I have exactly the same thing happening with Gauteng.

There is only weekly NICD reports and the NICD dashboard.
